src/OpenXR/Extensions/Silk.NET.OpenXR.Extensions.FB/FBSwapchainUpdateState.gen.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Silk.NET.Core;
+using Silk.NET.Core.Native;
+using Silk.NET.Core.Attributes;
+using Silk.NET.Core.Contexts;
+using Silk.NET.Core.Loader;
+using Silk.NET.OpenXR;
+using Extension = Silk.NET.Core.Attributes.ExtensionAttribute;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.OpenXR.Extensions.FB
+{
+    [Extension("XR_FB_swapchain_update_state")]
+    public unsafe partial class FBSwapchainUpdateState : NativeExtension<XR>
+    {
+        public const string ExtensionName = "XR_FB_swapchain_update_state";
+        /// <summary>To be documented.</summary>
+        [NativeApi(EntryPoint = "xrUpdateSwapchainFB")]
+        public unsafe partial Result UpdateSwapchainFB([Count(Count = 0)] Swapchain swapchain, [Count(Count = 0), Flow(FlowDirection.In)] SwapchainStateBaseHeaderFB* state);
```

Comment:
Just to note this, why is Count = 0 here? Consider referencing in new issue.

Reply:
Same as in all Vulkan and OpenXR bindings. `Count = 0` should be treated as there being no count at all.